2001-3000/LOC2313DabraM.xml

```diff
@@ -49,14 +49,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <body>
          <listPlace>
             <place type="monastery">
-               <placeName xml:lang="gez" xml:id="n1">ደብረ፡ ማርያም፡</placeName>
-               <placeName xml:lang="gez" corresp="#n1" type="normalized">Dabra Māryām</placeName>
-               
-               
-            
+               <placeName xml:lang="gez" xml:id="n1">ደብረ፡ ማርያም፡ ቈሐይን፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1" type="normalized">Dabra Māryām Qʷaḥayn</placeName>
+
```

Comment:
take Qwahayn out here, update 2317
check how we encode geographic position/location, e.g. , for 2313 it should be
```
 <country ref="LOC3010Ethiop"/>
               <region notBefore="1991" type="RegionalState" ref="LOC5932Tegray"/>               
               <settlement ref="LOC1298Akkala" type="warada"/>
```
for 2317 accordingly
```
 <country ref="LOC2991Eritre"/>
               <region ref="LOC5575Saraye"/>      
<settlement ref="LOC5271Qwahay"/>         
               
```

Reply:
ok